cmd/leo/read/read.go

```diff
@@ -62,39 +68,69 @@ func (o *Options) Run(cmd *cobra.Command, args []string, cloud *common.Cloud) er
 		return err
 	}
 
-	if err := tsk.Read(ctx); err != nil {
-		return err
-	}
+	for last := 0; ; {
+		if err := tsk.Read(ctx); err != nil {
+			return err
+		}
 
-	switch {
-	case o.Logs:
-		return o.printLogs(ctx, tsk)
-	case o.Status:
-		return o.printStatus(ctx, tsk)
-	}
+		logs, err := o.getLogs(ctx, tsk)
+		if err != nil {
+			return err
+		}
+
+		status, err := o.getStatus(ctx, tsk)
+		if err != nil {
+			return err
+		}
 
-	return nil
+		if delta := strings.Join(logs[last:], "\n"); delta != "" {
```

Comment:
This is a really complicated way of saying ` if len(logs) > last {`

Reply:
This accounts for cases where `logs := []string{""}` and an extra newline would be printed in the beginning. Are you sure it's equivalent? 🤔

Reply:
not sure I understand what you mean - are you trying to work around the case where empty log lines are appended?

Reply:
Sort of. It appears that the first returned value of `logs` is no t an empty slice, but a slice containing an empty string.

Reply:
I see, let's leave it as is then